### DISCUSSION TITLE
Granular user permissions

### DISCUSSION THREAD

**Discussion by husseinalhammad** [statamic/cms#8029] [Q&A]
Hi,

I have a use case that requires some granular user permissions. I would like to:

1. Allow a user to only edit specific entries in a collection. This user is not the author.
2. Allow a user to edit/delete entries in a collection only if a specific (Entries) field has a certain value.
3. Allow a user to add new entries to a collection, but restrict what value they can assign to a specific (Entries) field.
4. Allow a user to view form submissions only if a specific field received a certain value.

The [native user permissions](https://statamic.dev/users#permissions) are great, but don't get me exactly what I need. Is the above achievable with [custom permissions](https://statamic.dev/extending/permissions)?

---

This is the closest thread I found discussing something similar (https://github.com/statamic/cms/discussions/5594). I rather not create multiple collections/forms.

**Comment by edalzell** [comment]
> Is the above achievable with [custom permissions](https://statamic.dev/extending/permissions)?

Maybe, give it a try on one of them?

**Reply by husseinalhammad** [reply]
I would but user management is a pro only feature. This client is yet to agree to use Statamic and I have no other pro license to use to tinker with pro features.

**Reply by jackmcdade** [reply]
You don't need a license to try Pro: https://statamic.dev/licensing#trial-mode

**Reply by husseinalhammad** [reply]
Oh sweet! I didn't realise this. I'll update this thread after I look into this. Thank you both!
